scripts/org-setup.mjs

```diff
@@ -0,0 +1,84 @@
+import 'zx/globals'
+import { writeFile } from 'node:fs/promises'
+import { join } from 'node:path'
+import { EOL } from 'node:os'
+
+if (!process.env.SF_HUB_USERNAME || !process.env.SF_OAUTH2_JWT_KEY || !process.env.SF_OAUTH2_CLIENT_ID) {
+  throw new Error('Set SF_HUB_USERNAME, SF_OAUTH2_JWT_KEY and SF_OAUTH2_CLIENT_ID env vars.')
+}
+
+$.verbose = false
```

Comment:
disable zx printing output of executed commands:
https://github.com/google/zx#verbose